CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Public: Fix `CROSS_DEVICE_START` user analytic event for integrators never being dispatched when user switches to the Cross Device flow
 - UI: Update copy in Face Liveness Video intro screen from 25s to 20s to reflect the correct time limit
 - Remove old locale key type definitions that are no longer used/exist in code base.
+- UI: Fixed the text placement to be bellow the button, in the `CustomFileInput` component.
```

Comment:
Minor: I don't think the comma is needed.
```suggestion
- UI: Fixed the text placement to be bellow the button in the `CustomFileInput` component.
```

Reply:
Comma removed.